consensus_error.txt

```diff
@@ -0,0 +1,109 @@
+Incorrect ledger_hash hash for block 316000.  Calculated 50e6c784b35badf8ac41c5d933f5bff881f09569b09a99eb48947f7050808734 but expected 5c4506b2863161af45ef7f4afa0fe8730826a0e671cbdb3d9ecfde93d370c972
```

Comment:
Can we add some comments to indicate the context for these errors? (Also this should probably not be here.)

Reply:
oops! it's just a not relevant debug file! will remove it!